src/modules/IscsiLioServer.rb

```diff
@@ -169,6 +169,18 @@ def getServiceStatus
         @statusOnStart = true
         @serviceStatus = true
       end
+      status = File.exist?("/etc/target/lio_setup.sh")
+      if !status
+      SCR.Execute(path(".target.bash_output"), "touch /etc/target/lio_setup.sh")
+      SCR.Execute(path(".target.bash_output"), "chmod 755 /etc/target/lio_setup.sh")
+      SCR.Execute(path(".target.bash_output"), "echo '#'!/bin/bash > /etc/target/lio_setup.sh")
+      end
+      status = File.exist?("/etc/target/tcm_setup.sh")
+      if !status
+      SCR.Execute(path(".target.bash_output"), "touch /etc/target/tcm_setup.sh")
+      SCR.Execute(path(".target.bash_output"), "chmod 755 /etc/target/tcm_setup.sh")
+      SCR.Execute(path(".target.bash_output"), "echo '#'!/bin/bash > /etc/target/tcm_setup.sh")
```

Comment:
I haven't tested the code, so I don't know whether some other issues might arise, especially when config file doesn't exist and then appears - what happens when Yast then tries to write that file? I don't know.